pkg/cli/config_check.go

```
@@ -0,0 +1,150 @@
+	return configCheckReport{
+		SchemaVersion:        inspection.SchemaVersion,
+		CurrentSchemaVersion: inspection.CurrentSchemaVersion,
+		SchemaState:          inspection.SchemaState,
+		Valid:                inspection.SchemaState == config.SchemaStateCurrent && !inspection.HasErrors(),
+		AWS:                  aws,
+		Findings:             findings,
+	}
+}
+
+func printConfigCheckReport(out io.Writer, report configCheckReport) {
+	fmt.Fprintln(out, "🔎 Checking .kit.yaml...")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)